src/lib/api.js

```diff
@@ -157,3 +159,11 @@ export function get(path, options = {}) {
     return checkResponseStatus(response);
   });
 }
+
+export function getGithubRepos(token) {
+  return fetch(`/api/github-repositories?access_token=${token}`)
+    .then(checkResponseStatus)
+    .then(repos => {
```

Comment:
I would move the filter on the `OpenSourceApplyPage` side, to keep `getGithubRepos` generic.

Reply:
Actually I can remove it completely since have added to the backend.